addon/components/au-pill.hbs

```diff
@@ -17,7 +17,7 @@
       {{/if}}
     {{/if}}
   </span>
-  <button class="au-c-pill-action {{this.actionSize}} {{this.skin}}" type="button" {{@action}}>
+  <button class="au-c-pill-action {{this.actionSize}} {{this.skin}}" type="button" {{on "click" @onClickAction}}>
```

Comment:
> If we're renaming I would drop the Action suffix and make it `@onClick` instead? That seems more in line with how it's normally done since "action" is implied.

I think this one could be easily done both ways, but I chose `@onClickAction` because it is specifically bound the the action attached to the pill. `@onClick` would the suggest it applies to the whole pill? @Dietr Do you have a take on this?

Reply:
@Windvis Regarding the code coverage: I added a test for the action click. Can you take a peek and let me know if this is ok (first time working with QUnit)?

https://github.com/appuniversum/ember-appuniversum/pull/270/files#diff-41d099abcafacca8df9533840f31571d874dd2fc525d2b5864d0ddbdc59e2543

Reply:
> I think this one could be easily done both ways, but I chose @onClickAction because it is specifically bound the the action attached to the pill. @onClick would the suggest it applies to the whole pill? @Dietr Do you have a take on this?

Ah, I did indeed think the user would click the whole pill. I assumed the `Action` was refering to Ember's concept of "actions". My bad, I'm unfamiliar with the component 😅.

In that case it might make sense, but also think something like`@onActionClick` sounds better 😄. (Similar to how I would go for `@onButtonClick` instead of `@onClickButton`). But that's probably just a preference thing so I'll let you and Dieter decide on that 👍.

Reply:
And the test looks fine, thanks!

Reply:
> Ah, I did indeed think the user would click the whole pill. I assumed the `Action` was refering to Ember's concept of "actions". My bad, I'm unfamiliar with the component 😅.
> 
> In that case it might make sense, but also think something like`@onActionClick` sounds better 😄. (Similar to how I would go for `@onButtonClick` instead of `@onClickButton`). But that's probably just a preference thing so I'll let you and Dieter decide on that 👍.

No problem!

Regarding `@onClickAction`: I wasn't really sure what to pick here because I also found `@onClickRow` inside the codebase, but I have to admit that the other way around sounds a tad better. @Dietr, if you can find the time, what is your take?